### DISCUSSION TITLE
Ansible Role to install, configure and manage the Advanced Intrusion Detection Environment (AIDE)

### DISCUSSION THREAD

**Discussion by Tronde** [linux-system-roles/linux-system-roles.github.io#103] [Ideas]
Dear Team,  
I have used the [linux-system-roles/template](https://github.com/linux-system-roles/template) to create an Ansible role that is able to install, configure, and manage the [Advanced Intrusion Detection Environment (AIDE)](https://aide.github.io/). The role itself you can find at URL: [https://github.com/Tronde/aide](https://github.com/Tronde/aide)

Do you think this role would be a good addition for the linux-system-roles project?

If you find the time to look into this role and provide some feedback on it, it would be much appreciated.

Please let me know your thoughts.

Best regards,
Joerg

**Comment by spetrosi** [comment]
Hi @Tronde, this was on our radars for some time, we ended up adding a repo in our namespace and using your role as a basis in https://github.com/linux-system-roles/aide/pull/3. It was easier to apply our template and use our CI this way.
I'll close this issue because the initial version of the role has been merged. Some tests are to be added.

If you have time, could you review the initial version of the role?

Thank you

**Reply by Tronde** [reply]
Hi @spetrosi, that's great news. I'm happy that my role was useful to some degree and is a basis in the new system role.

Currently I have no spare time to test it, but that will change in December. Than I'm happy to give it a try. :-)
